solidity/contracts/converter/ConverterUpgrader.sol

```diff
@@ -165,10 +171,6 @@ contract ConverterUpgrader is IConverterUpgrader, ContractRegistryClient {
             if (reserveAddress == ETH_RESERVE_ADDRESS) {
                 _newConverter.addReserve(ETH_RESERVE_ADDRESS, weight);
             }
-            // Ether reserve token
-            else if (reserveAddress == etherToken) {

```

Comment:
Do we need to have `EtherToken` as part of the repo?

Reply:
I have asked the same question.
It is still being used at the `BancorNetwork` level.
And generally speaking - this change is unrelated of the PR (as you may have realized by now).

Reply:
let's open a separate task for this

Reply:
Move this fix out of this branch?
Or just fix the `BancorNetwork` level in a new branch?

Reply:
leave the branch as is and open another task for it (not to be fixed now)